enhancements/installer/openstack-for-power-vc-ipi.md

```diff
@@ -0,0 +1,314 @@
+the graduation criteria.
+- `Tech Preview` will graduate to `GA` in OpenShift 4.22
+- Our current target for `Tech Preview` is OpenShift 4.21, but this will be reevaluated as work progresses. Commitments
+are tracked in the Multi-Arch program sign-off documents for each release, which is created as part of Quarterly
+Planning.
+- [Deprecation policy][deprecation-policy]
+
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+### Dev Preview -> Tech Preview
+
+- [ ] Full functionality / ability to utilize the enhancement end to end
+- [ ] End-user documentation
+- [ ] CI e2e jobs created to cover installation against this platform
+- [ ] Plan defined & support engaged to gather feedback from users
+
+### Tech Preview -> GA
+- [ ] CI has been upgraded to cover testing of modifications to the installer (PR validation)
+- [ ] CI "soak" for to ensure a stable baseline
+- [ ] Final optimizations for install time
+- [ ] Expanded testing (upgrade, downgrade, scale, load/performance)
+- [ ] Documentation is finalized
+
+### Removing a deprecated feature
+
+- Announce deprecation and support policy of the existing feature
+- Deprecate the feature
+
+## Upgrade / Downgrade Strategy
+
+All upgrade and downgrade expectations are the same for existing IPI deployments.
+
+## Version Skew Strategy
+
+This should be the same as other IPI deployments.
+
+## Operational Aspects of API Extensions
+N/a
+
+#### Failure Modes
+N/a
+
+## Support Procedures
+N/a
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation History`.
+
+## Drawbacks
+
+## Alternatives (Not Implemented)
+
+We can continue to rely on our UPI playbooks but this is not ideal as they are not supported. We can also point users to
+our assisted installer support for PowerVC, but we believe users desire a more automated solution in this environment.
+
+Adding code paths specific to PowerVC to the OpenStack provider was explored. For example, we tried adding the ability
+to skip security group creation when using powervc. The OpenStack team was not open to accepting this as they didn't want
+to imply that all OpenStack clusters should consider not using security groups. The changes needed to address the
+differences between OpenStack and PowerVC would not benefit many users of the OpenStack IPI plaform. Because of this, we
+decided that the best way to isolate these changes would be to create a shim provider that handled the differences in
+environment while still leveraging the existing OpenStack IPI platform.
+
+Using the external provider was another option that we explored. PowerVC is unique in that it is a variant of
+OpenStack and would not need as much customization as the external provider enables. Additionally, we felt that the
+user experience would be easier if powervc was its own platform.
```

Comment:
Also, I am not sure if we can use an exising in-tree provider (Openstack) and call it external.